config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+Kaminari.configure do |config|
+  config.default_per_page = 10
```

Comment:
The config looks good!

Since our wishlist item partials are so large, could you please set the default to 5?